Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/events/DefaultTraceLocation.kt

```diff
@@ -0,0 +1,45 @@
+package de.rki.coronawarnapp.eventregistration.events
+
+import de.rki.coronawarnapp.eventregistration.storage.entity.TraceLocationEntity
+import org.joda.time.Instant
+
+const val TRACE_LOCATION_VERSION = 1
+
+data class DefaultTraceLocation(
+    override val guid: String,
+    override val type: TraceLocation.Type,
+    override val description: String,
+    override val address: String,
+    override val startDate: Instant?,
+    override val endDate: Instant?,
+    override val defaultCheckInLengthInMinutes: Int?,
+    override val signature: String,
+    override val version: Int = TRACE_LOCATION_VERSION,
+) : TraceLocation
+
+fun List<TraceLocationEntity>.toTraceLocations() = this.map { it.toTraceLocation() }
+
+fun TraceLocationEntity.toTraceLocation() = DefaultTraceLocation(
```

Comment:
Please add parameter names for calls with so many parameters, especially with many being of the same type, to prevent someone in the future from accidentally swapping some.

Reply:
Maybe add a unit test for this.

Reply:
Added parameter names and test in a new commit